### DISCUSSION TITLE
notify_status() Raises NotFoundOnGithub Error

### DISCUSSION THREAD

**Discussion by yibudak** [sbidoul/runboat#121] [Deployment]
Hello,

When attempting to use the runboat on private repositories, I encounter an error in the `notify_status()` function. However, other GitHub functions, such as `get_pull_info()` and `get_branch_info()`, work as expected.

Which token type should be utilized in the `github_token` parameter? The documentation provides an example of a fine-grained token, but when I employ this token, the functions above also fail.

Create a commit status (docs):
https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28#create-a-commit-status

**Comment by yibudak** [comment]
I may have found a solution. First, I tried allowing tokens (classic and fine-grained) in the organization, but it didn't change anything. Making the bot an owner of the organization solved the problem. Also I have to mention that I made it work with classic token, maybe we can add more information about it in the code.

**Comment by sbidoul** [comment]
The token must be for a user with push permissions to the repos. I don't think it is necessary for it to be a org owner.
